src/main/java/com/minecraftabnormals/abnormals_core/core/util/DataUtil.java

```diff
@@ -174,24 +179,6 @@ public static boolean matchesKeys(ResourceLocation loc, RegistryKey<?>... keys)
 		return false;
 	}
 
-	/**
-	 * <p>Registers a {@link IDispenseItemBehavior} that will perform the new behavior if the condition is met and the behavior that was already in the registry if not.
-	 * This works even if multiple mods add new behavior to the same item.</p>
-	 * <p>Ideally, the condition should be implemented such that the predicate only passes if the new behavior will be 'successful', avoiding problems with failure sounds not playing.</p>
-	 *
-	 * @param item The {@link Item} to register the {@code newBehavior} for.
-	 * @param condition A {@link BiPredicate} that takes in {@link IBlockSource} and {@link ItemStack} arguments, returning true if the {@code newBehavior} should be performed.
-	 * @param newBehavior The {@link IDispenseItemBehavior} that will be used if the {@code condition} is met.
-	 *
-	 * @author abigailfails
-	 */
-	public static void registerAlternativeDispenseBehavior(Item item, BiPredicate<IBlockSource, ItemStack> condition, IDispenseItemBehavior newBehavior) {
```

Comment:
The removal of this method is a breaking change.
It should be kept and integrated with this new system, but marked as deprecated.

Reply:
Readded in latest commit.